stress_test.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+"""
+Stress Test Script
+
+This script performs load testing on the eDrop API endpoint by sending concurrent requests
+at a specified rate for a specified duration.
```

Comment:
Let's add a warning here that this should be run when mocking REDCap as well as GBF.